Gems/Atom/RPI/Code/Source/RPI.Public/RPIUtils.cpp

```diff
@@ -113,12 +114,22 @@ namespace AZ
                 case AZ::RHI::Format::A8_UNORM:
                 case AZ::RHI::Format::R8G8_UNORM:
                 case AZ::RHI::Format::R8G8B8A8_UNORM:
+                case AZ::RHI::Format::A8B8G8R8_UNORM:
                 {
                     return mem[index] / static_cast<float>(std::numeric_limits<AZ::u8>::max());
                 }
+                case AZ::RHI::Format::R8_UNORM_SRGB:
+                case AZ::RHI::Format::R8G8_UNORM_SRGB:
+                case AZ::RHI::Format::R8G8B8A8_UNORM_SRGB:
+                case AZ::RHI::Format::A8B8G8R8_UNORM_SRGB:
+                {
+                    float srgbValue = mem[index] / static_cast<float>(std::numeric_limits<AZ::u8>::max());
+                    return AZ::Color::ConvertSrgbGammaToLinear(srgbValue);
```

Comment:
Suggestion: It might be worth having a second version of the conversion function that just has a static 256-value lookup table that takes a uint8 and returns a float. There's a lot of math happening here for something that could be precomputed.

Reply:
I can do that!